packages/scene-composer/src/utils/controlUtils.ts

```diff
@@ -0,0 +1,36 @@
+import { CameraControlsType } from '../interfaces';
+
+export const isPointerLockControl = (controlType: CameraControlsType): boolean => {
+  if (controlType === 'pointerLock') {
```

Comment:
i <3 ternary
```
return controlType === 'pointerLock'
 ? true
 : false

Reply:
Okay converting this and the below to one line ternary